### ISSUE TITLE
Trigger Jobs Manually

### ISSUE THREAD

**Issue by tanweerdev** [quantum-elixir/quantum-core#452]
something like 
```new_job()
    |> Quantum.Job.set_name(:test)
    |> Quantum.Job.set_schedule(:asap or :now)
    |> Quantum.Job.set_task(fn ->
      IO.puts("#{Timex.now()} - first hello")
    end)
    |> add_job()
```

https://github.com/quantum-elixir/quantum-core/issues/392

**Comment by maennchen** [comment]
Currently this is not possible.

Why would you want to do this via quantum? Wouldn‘t Job.async or similar work for you?

**Comment by tanweerdev** [comment]
Job.async is part of some library? which one?

**Comment by tanweerdev** [comment]
I am getting this warning and timeout inside a channel event handle_in method/3
`[warn] Phoenix.Channel.Server #PID<0.765.0> received unexpected message in handle_info/2: {#Reference<0.4027534044.3015442433.81688>, {:ok, %ChannelConnection{__meta__: #Ecto.Schema.Metadata<:loaded, "channel_connections">, ...updated_at: ~N[2020-08-24 11:49:10.057884]}}}`
and the documentation says `Starts a task that must be awaited on.` but I dont want to wait for this task to finish. This should continue totally independently...

**Comment by tanweerdev** [comment]
@maennchen I mean `now` is also a timestamp and quantum should be able to handle it

**Comment by maennchen** [comment]
@tanweerdev You can also run it completely independently with Task. Have a look at its functions.

Quantum currently only works with cron expressions. Opening that functionality up for different methods would be a long time goal (#268). Just adding a single atom as a schedule is however not a direction I‘d lile to go.

Quantum is about scheduling tasks i. intervals and i. the future. For right now there’s a lot of other options.

**Comment by ericdude4** [comment]
Quantum handles event overlap, so it would be nice to be able to call `MyApp.Scheduler.run_job(:my_job)` and then have the overlap prevention stop this "one-off" job from overlapping with the normal job schedule.

**Comment by maennchen** [comment]
@ericdude4 That sounds like a reasonable enhancement. A PR would be very welcome.

**Comment by ericdude4** [comment]
@maennchen I would be happy to

**Comment by maennchen** [comment]
@ericdude4 Great! If you have any questions, I'll be happy to provide some guidance. (Here or on #quantum in the elixir slack.)

**Comment by maennchen** [comment]
Released as `v3.3.0`
